### ISSUE TITLE
CUDA Training Option

### ISSUE THREAD

**Issue by Berk035** [BLG638-Project/final-2021#8]
Hello everyone,

1) We've realized that the line in /sac_example/evaluate.py line 22:

`device = torch.device("cuda" if torch.cuda.is_available() else "cpu")`

does not assign "cuda" altough cuda is avaiable. We've checked that avalibility in environment with:
"import torch / torch.cuda.is_available()" and it returns True. Unfortunately, it is False in the simulation code. Is there any solution for this? 

2) Also, Can we use images of the road or simulation windows as observation? For instance, I want to collect [64]*[64]*[3] images. Is this possible ?

Thanks in advance.

**Comment by ferhatmelih** [comment]
Hi
1. it is your responsibility to define training in CPU/GPU. along with evaluation. The provided example may not use this as an argument. your model can. Why should this be a problem? 
2. No. Environment oberservation is fxed. you can change reward configuration but not I/O of the environment.

**Comment by Berk035** [comment]
Thank you for your answer. I've knew that it is our responsibility but, that line contains "cuda", so I've just wanted to ask to reducing training time. Have a good day.
